docs/docs/airs/airs_doc.md

```diff
@@ -0,0 +1,166 @@
+# ARLAS Item Registration Services (AIRS)
+
+ARLAS Item Registration Services offers registration services for Spatio-temporal assets. It manages Items as well as Assets (e.g. raster files, cogs, etc.). 
+
+The service exposes the STAC-T methods [github.com/stac-api-extensions/transaction](https://github.com/stac-api-extensions/transaction) as well as a set of methods for handling the assets.
+
+By default, the service manages the assets. When an item is registered, the service checks that the managed asset exists: the asset must be added before the item. Deleting an item is cascaded on the managed assets. An asset can be unmanaged by setting `asset.airs:managed=False` (or `asset.airs__managed=False`).
+
+## AIRS Data model
+
+The AIRS Model is based on the STAC specifications. It supports the following extensions:
+
+- view
+- storage
+- eo
+- processing
+- dc3 (ARLAS Datacube Builder)
+- cube
+- sar
+- proj
+
+Also, metadata are enriched by the service and placed in the `generated` namespace.
+
+Namespaces are prefixes in the key names of the JSON. The `:` is used for separating the namespace and the field name. 
+
+!!! warning
+    Since ARLAS does not support the `:` in field names, the character is replaced by `__` for storage and indexation.
+
+For more details, see the [AIRS API documentation](airs_api.md)
+
+
+## Using AIRS
+
+In the following examples, we will:
+
+- add an asset
+- check that it exists
+- add an item
+- get the item
+- delete the item and its asset
+
+### Prerequisites
+
+- minio
+- elasticsearch
+- docker
+
+See [here](https://hub.docker.com/r/gisaia/airs/tags){:target="_blank"} for the available versions of airs.
+
+!!! note
+    The service is deployed on `ARLAS_ENDPOINT` 
+
+### AIRS Configuration
```

Comment:
this `AIRS Configuration` should go to the project readme, not the doc that goes to docs.arlas.io